### PR TITLE
Documentation for the new --log-output=file option

### PR DESCRIPTION
**Pull request by codebien**
Added the documentation for https://github.com/grafana/k6/pull/2285 that has added the support for the log files.

This PR is expected to be merged when the `k6` v0.37.0 version is going to be released.

### CODE REVIEW

**Comment by codebien** [comment]
@MStoykov @oleiade I assigned to you because you reviewed the PR.

File: src/data/markdown/translated-guides/en/02 Using k6/05 Options.md [hunk 1]
```diff
@@ -695,45 +695,74 @@ $ k6 run --local-ips=192.168.20.12-192.168.20.15,192.168.10.0/27 script.js
...
+$ k6 run --log-output=file=./k6.log script.js
 ```
 
 </CodeGroup>
 
+A valid file path is the unique mandatory field, the other optional fields listed below:
+
+| key | description | default value |
+| --- | ----------- | ------------- |
+| level | the minimal level of a message to write | all |
```
Comment (MStoykov) [inline src/data/markdown/translated-guides/en/02 Using k6/05 Options.md:764]:
```suggestion
| level | the minimal level of a message to write out of (in ascending order): trace, debug, info, warning, error, fatal, panic | trace |
```

**Review by MStoykov — APPROVED** [review]
Small note on the actual possible levels

**Review by MStoykov — APPROVED** [review]

**Review by oleiade — APPROVED** [review]
🎉 🦖 🌮

**Review by olegbespalov — APPROVED** [review]

**Pull request merged**